src/service/extension-location.js

```diff
@@ -105,10 +115,37 @@ export function calculateEntryPointScriptUrl(
 export function parseExtensionUrl(scriptUrl) {
   // Note that the "(\.max)?" group only applies to local dev.
   const matches = scriptUrl.match(
-    /^(.*)\/(.*)-([0-9.]+|latest)(\.max)?\.(?:js|mjs)$/i
+    /^(.*)\/(.*)-([0-9.]+|latest)(\.max)?\.(?:js|mjs|sxg\.js)(\?.*)?$/i
```

Comment:
What I meant here is that we don't need to use `f=sxg` anymore. The point of `f=sxg` was to avoid changing the runtime regex, but we had to change it anyways. So why not just go with `.sxg.js` everywhere?